MinimedKit/PumpEvents/PumpEvent.swift

```diff
@@ -21,3 +21,16 @@ public protocol PumpEvent : DictionaryRepresentable {
     }
     
 }
+
+public extension PumpEvent {
+    public func canBeDelayedAppend() -> Bool {
```

Comment:
We're encountering a type smell here

PumpModel is required for the initializer as defined in the PumpEvent protocol

Given that, we shouldn't pass it in again on an extension method.

My sense is that a PumpEvent shouldn't care about PumpModel. But some of the PumpEvent subclasses do and that is currently baked into the design.